iac/key_vault.tf

```diff
@@ -0,0 +1,39 @@
+data "azurerm_client_config" "current" {}
```

Comment:
Is this to query the current subscription?

Reply:
I believe so, yes.

https://www.terraform.io/docs/providers/azurerm/d/client_config.html